src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -38,10 +40,16 @@ def environment_variables_format(value):
 register_cli_argument('container create', 'memory', type=float, help='The required memory of the containers in GB')
 register_cli_argument('container create', 'os_type', help='The OS type of the containers', **enum_choice_list(OS_TYPES))
 register_cli_argument('container create', 'ip_address', help='The IP address type of the container group', **enum_choice_list(IP_ADDRESS_TYPES))
+register_cli_argument('container create', 'ports', type=int, nargs='+', help='The ports to open')
+register_cli_argument('container create', 'restart_policy', help='Restart policy for all containers within the container group', **enum_choice_list(RESTART_POLICIES))
```

Comment:
You should just be able to do `**enum_choice_list(ContainerGroupRestartPolicy)` right?
The idea is that if `ContainerGroupRestartPolicy` updates with a future SDK version, you don't have to manually add to the list yourself.